bin/make-release.sh

```diff
@@ -43,10 +43,16 @@ find vendor/ -type f -name "*.php" \
  | xargs -L1 git add -f
 find vendor/ -type f -name LICENSE | xargs -L1 git add -f
 find vendor/ -type f -name '*.json' | xargs -L1 git add -f
-find asset/ -type f | xargs -L1 git add -f
+find -L asset/ -type f | xargs -L1 git add -f
 echo "v$VERSION" > VERSION
 git add VERSION
 git add composer.lock -f
+cat <<gitattributes >>.gitattributes
+.git* export-ignore
+bin/ export-ignore
+RELEASE.md export-ignore
+gitattributes
+git add .gitattributes
```

Comment:
Why is this not a file in git?

Reply:
I thought it was okay to still have them when archives are being created from the master branch.

Reply:
Why should anyone do so? The readme states that the master branch is not usable. And if one wants to create a new release, publishing it requires git access anyway.